src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Aragon Network Dashboard</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no, maximum-scale=1, user-scalable=no"
+    />
+
+    <!-- Provide fallback favicon.png for browsers that don't support .svg format -->
+    <link rel="icon" type="image/svg+xml" href="icons/favicon.svg" />
+    <link rel="icon" type="image/png" href="icons/favicon.png" />
```

Comment:
(Not necessarily in this PR)

We might want to use a higher size than 33x33 for the PNG, maybe 128x128? The SVG should be fine at any size, but it could happen that the SVG size is used to produce the raster image at the same size, so I think we could use 128x128 for both?

Reply:
Icon compat is a bit of a minefield :( but this is a good suggestion! Adri is adjusting some graphics too so i'll do them at the same time separately :)